cmd/certsuite/pkg/claim/claim.go

```diff
@@ -83,7 +83,7 @@ type Schema struct {
 		Nodes Nodes `json:"nodes"`
 
 		RawResults struct {
```

Comment:
I believe the RawResults section was removed from the claim file [here](https://github.com/redhat-best-practices-for-k8s/certsuite-claim/pull/111). We should probably review these structs and and the `certsuite claim show/compare` subcommands in further PRs.